### PR TITLE
updated as suggested

### PR DESCRIPTION
**Pull request by brianwmao**
updated as suggested

### CODE REVIEW

File: cc_dv_bm3024.Rmd [hunk 1]
```diff
@@ -0,0 +1,190 @@
+# Plotly R graphing
+
+Brian Mao
+## Introduction
+
+Plotly's R graphing library makes interactive, publication-quality graphs. in this into to Plotly I will show some examples of how to make line plots, scatter plots, area charts, bar charts, box plots, histograms, heatmaps, and 3D scatter plot charts. there are a lots more advanced function in the Plotly. you may explore it later on yourself 
+
+```{r setup, include=FALSE}
```
Comment (shauryamalik) [inline cc_dv_bm3024.Rmd:8]: Please follow 2.2.6 (https://jtr13.github.io/cc21/github-submission-instructions.html) and remove 'setup'

File: cc_dv_bm3024.Rmd [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+library(plotly)
+fig <- plot_ly(data = iris, x = ~Sepal.Length, y = ~Petal.Length, type = 'scatter',
+  mode = 'markers', symbol = ~Species, symbols = c('circle','x','o'),
+  color = I('black'), marker = list(size = 10))
+fig
+```
+
+## Statistical Charts
+
+#### Basic Boxplot
```
Comment (shauryamalik) [inline cc_dv_bm3024.Rmd:63]: On line 61 the heading has 2 hashtags, line 63 must have 3 hashtags for correct ordering.

File: cc_dv_bm3024.Rmd [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+                     zaxis = list(title = '1/4 mile time')))
+
+fig
+```
+
+## Conclusion
+
+In this project I explored different methods of plotting interactive plot by utilizing plotly and ggplot2. you can explore more about the tool using R, python and Javascript. there are more resource available below. it is a very user friendly tool and the interface is well designed. it  allows users to design their own way to tell the story to its audience. effectively utilizing this tool will tremendously help you to improve your data visualization skills
+
+## 8 Work Cited:
```
Comment (shauryamalik) [inline cc_dv_bm3024.Rmd:154]: Number is not needed with the hashtags.

**Review by shauryamalik — CHANGES_REQUESTED** [review]

**Comment by shauryamalik** [comment]
There is no need to create a new PR every time you make changes. You can continue to edit these files by committing to the same branch. Furthermore if https://github.com/jtr13/cc21/pull/94 is a duplicate PR please close it.

**Comment by brianwmao** [comment]
Hi Shaurya,

I am able to do the pull request anymore would you please upload my file
please

please find attached file

Thank,
Brian

**Pull request closed without merging**